new_mvc/controllers/authController.js

```diff
@@ -41,47 +41,49 @@ exports.validateUser = async (req, res) => {
 
         const [row] = await auth.validateUser(req.body.login);
 
+        const dbUser = row[0];     //  !!!!!
+
         console.log("row:", row);
```

Comment:
Here and in all pull request - remove `console.log`. They are needed only in development